alchemy-web/src/content/docs/guides/cloudflare.mdx

```diff
@@ -7,272 +7,79 @@ sidebar:
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
+Authentication is configured using the [`alchemy configure`](/concepts/cli#configure) command. 
+
 There are three supported ways of authorizing Alchemy with Cloudflare:
-1. API Token - a token you create once with limited scopes
-2. OAuth - a token created by `wrangler login`
+1. OAuth - a token created by oauth with cloudflare
+2. API Token - a token you create once with limited scopes
 3. Global API Key (legacy) - the global, highly permissive API key
 
-## API Token
-
-First you need to [create an API Token](https://developers.cloudflare.com/fundamentals/api/get-started/create-token/) and then use it in your Alchemy app.
-
-By default, Alchemy will use the `CLOUDFLARE_API_TOKEN` environment variable if set.
+## OAuth
 
-You can store the token in your `.env` file
-```sh
-CLOUDFLARE_API_TOKEN=<token>
-```
-
-Or set when deploying your app:
-
-<Tabs syncKey="pkgManager">
-  <TabItem label="bun">
-    ```sh
-    CLOUDFLARE_API_TOKEN=<token> bun alchemy deploy
-    ```
-  </TabItem>
-  <TabItem label="npm">
-    ```sh
-    CLOUDFLARE_API_TOKEN=<token> npx alchemy deploy
-    ```
-  </TabItem>
-  <TabItem label="pnpm">
-    ```sh
-    CLOUDFLARE_API_TOKEN=<token> pnpm alchemy deploy
-    ```
-  </TabItem>
-  <TabItem label="yarn">
-    ```sh
-    CLOUDFLARE_API_TOKEN=<token> yarn alchemy deploy
-    ```
-  </TabItem>
-</Tabs>
-
-You can explciitly set an `apiToken` when creating a Cloudflare Resource, such as a `Worker`:
-
-```ts
-await Worker("my-worker", {
-  apiToken: alchemy.secret(process.env.MY_TOKEN)
-});
-```
+1. Run `alchemy configure` and enter a profile name (or use the default profile).
+2. Select "OAuth".
+3. If you want to customize the scopes, select "Customize scopes", and select the desired scopes
+4. A browser window will open prompting you to authorize alchemy with Cloudflare.
+5. Once authorized return to the terminal and select the cloudflare account you want to use.
 
-:::caution
-To use `alchemy.secret`, you must set a `password` when initializing your alchemy app. See [Encryption Password](/concepts/secret#encryption-password).
+:::info
+By default alchemy will configure OAuth tokens with the same scopes as [wrangler](https://developers.cloudflare.com/workers/wrangler/).
 :::
 
-## OAuth Token
```

Comment:
I think we still support these as overrides.

Reply:
sorry what is this referencing? the old methods don't work because they are login based and we no longer have the old wrangler-based login command.

the ability to use env vars is mentioned at the bottom of the page